### DISCUSSION TITLE
Welcome to bdshare Discussions!

### DISCUSSION THREAD

**Discussion by rochi88** [rochi88/bdshare#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mgnbdl** [comment]
It seems a good tool for DSE stock data collection. But I do not know how to use it.

**Reply by rochi88** [reply]
Thank you for your comment. Please read the readme section for usage guideline. Thanks.
